backend/runner/taskrun/scheduler.go

```diff
@@ -355,9 +355,9 @@ func (s *Scheduler) PatchTask(ctx context.Context, task *store.TaskMessage, task
 	// Reset rollbackStatement and rollbackError because we are trying to build
 	// the rollbackStatement again and there could be previous runs.
 	if taskPatch.RollbackEnabled != nil && *taskPatch.RollbackEnabled {
-		empty := ""
-		taskPatch.RollbackStatement = &empty
-		taskPatch.RollbackError = &empty
+		var ptrStr *string
+		taskPatch.RollbackStatement = &ptrStr
```

Comment:
This looks a bit strange... the pointer of a pointer doesn't lool right.

Reply:
Change **string to *sql.NullString. PTAL